src/monitoring_tests/buffers_monitoring_test.py

```diff
@@ -0,0 +1,99 @@
+"""
+Checks the value of buffers every 150 settlements by invoking
+the ehtplorer api, and in some cases, coingecko.
+"""
+# pylint: disable=logging-fstring-interpolation
+import requests
+from src.monitoring_tests.base_test import BaseTest
+from src.apis.coingeckoapi import CoingeckoAPI
+from src.apis.klerosapi import KlerosAPI
+from src.constants import (
+    BUFFER_INTERVAL,
+    header,
+    REQUEST_TIMEOUT,
+    BUFFERS_VALUE_USD_THRESHOLD,
+)
+
+
+class BuffersMonitoringTest(BaseTest):
+    """
+    This test checks the value of the settlement contract buffers
+    every 150 settlements and generates an alert if it is higher than 200_000 USD.
+    Price feeds from ethplorer and coingecko (as backup) are used.
+    """
+
+    def __init__(self) -> None:
+        super().__init__()
+        self.coingecko_api = CoingeckoAPI()
+        self.kleros_api = KlerosAPI()
+        self.counter: int = 0
+
+    def compute_buffers_value(self) -> bool:
+        """
+        Evaluates current state of buffers.
+        """
+        # get all token balances of the smart contract
+        try:
+            ethplorer_data = requests.get(
+                "https://api.ethplorer.io/\
+                    getAddressInfo/\
+                    0x9008D19f58AAbD9eD0D60971565AA8510560ab41?\
+                    apiKey=freekey",
+                headers=header,
+                timeout=REQUEST_TIMEOUT,
+            )
+            ethplorer_rsp = ethplorer_data.json()
+            kleros_list = self.kleros_api.get_token_list()
```

Comment:
Handle the case of empty kleros_list here somehow. For example
```suggestion
            kleros_list = self.kleros_api.get_token_list()
            if kleros_list is None:
                kleros_list = []
```
or maybe just skip the hash?